package/MDAnalysis/analysis/diffusionmap.py

```diff
@@ -364,10 +365,191 @@ def transform(self, n_eigenvectors, time):
         time : float
             Exponent that eigenvalues are raised to for embedding, for large
             values, more dominant eigenvectors determine diffusion distance.
-        Return
-        ------
+
+
+        Returns
+        -------
         diffusion_space : array (n_frames, n_eigenvectors)
             The diffusion map embedding as defined by [Ferguson1]_.
         """
-        return (self._eigenvectors[1:n_eigenvectors+1,].T *
+        return (self._eigenvectors[1:n_eigenvectors+1, ].T *
                 (self.eigenvalues[1:n_eigenvectors+1]**time))
+
+    def plot_animated_transform(self, n_time=25, colorscale='Viridis', marker_size=3,
+                                x=1, y=2):
+        """ Plots an interactive 2D plot of chosen eigenvectors at different timescales.
+
+        Parameters
+        ----------
+        n_time: int, optional
+            number of time values for ``transform()``. Default: 25
```

Comment:
I think the default parameters are automatically included in the documentation from the function definition. Should we avoid specifying them in the docstring as well, in order to avoid possible divergences between documentation and code?

Reply:
Yes, I reckon so. I think I just do it because [everyone](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings) [else](https://scikit-learn.org/stable/modules/generated/sklearn.decomposition.PCA.html) [seems](https://numpy.org/doc/1.18/reference/generated/numpy.trace.html#numpy.trace) [to](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.apply.html#pandas.DataFrame.apply), but that's not a very good reason -- maybe it's from when `**kwargs` was everywhere.